webapp/tests/karma/ts/services/telemetry.service.spec.ts

```diff
@@ -91,13 +91,14 @@ describe('TelemetryService', () => {
 
   beforeEach(() => {
     defineWindow();
-    metaDb = {
+    db = {
       info: sinon.stub(),
       put: sinon.stub(),
       get: sinon.stub(),
-      query: sinon.stub()
+      query: sinon.stub(),
+      allDocs: sinon.stub()
     };
-    dbService = { get: () => metaDb };
+    dbService = { get: () => db };
```

Comment:
I think the "get" should be a stub that, depending of passed parameters (empty or { meta: true }), should return one of two DB mocks and we should make sure the queries are done against the correct db.

Reply:
That makes sense. Let me make the change.

Reply:
I have pushed a commit that separates the DB mocks. Didn't find a way to assert the queries are made against the correct db but hopefully having them separate makes it clear which db is being used. Please have a look and let me know what you think.

Reply:
Making them separate is exactly what I meant! Great! Thanks for making the change.